crates/rpc/src/bin/metric_exporter/metrics.rs

```diff
@@ -169,7 +163,8 @@ impl Metrics {
         *guard = Some(self.games_seconds.start_timer());
     }
 
-    pub fn end_game(&self) {
+    pub fn end_game(&self, remaining_players: i64) {
+        self.players_current.sub(remaining_players);
```

Comment:
Should this be `set`? given that we are talking about the remaining players
```suggestion
        self.players_current.sub(remaining_players);
```

Reply:
No; players current is supposed to be the current count of players; as a player joins, we increment it, and as a player disconnects we decrement it; but if a game ends, there might be remaining players that never disconnected so we subtract that amount.

In theory, of were doing one metric server per game, we could just do set(0) on startup, provided the stats aggregation will *sum* the amounts across each metrics server